geoips/testing/context_manager.py

```diff
@@ -0,0 +1,17 @@
+"""Module for handling optional dependencies throughout GeoIPS."""
+
+import logging
+from contextlib import contextmanager
+
+LOG = logging.getLogger(__name__)
+
+
+@contextmanager
+def import_optional_dependences(importer):
```

Comment:
`dependences` is misspelled and should be `dependencies`. Can you please update this everywhere?